server/app/views/applicant/ApplicantBaseFragment.html

```diff
@@ -39,21 +39,19 @@
   <span class="usa-hint--required">*</span>
 </th:block>
 
+<!-- The param is an instance of the ProgressBar class. -->
 <div
```

Comment:
This should probably have an aria-role="[progressbar](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/progressbar_role)".

Reply:
Will do. I added another "Done when" thing to https://github.com/civiform/civiform/issues/8847